src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -2,26 +2,55 @@
 
 import com.kodilla.ecommercee.domain.TokenGenerator;
 import com.kodilla.ecommercee.domain.UserDto;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
+import com.kodilla.ecommercee.domain.UserNotFoundException;
+import com.kodilla.ecommercee.mapper.UserMapper;
+import com.kodilla.ecommercee.service.UserDbService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
+@CrossOrigin(origins = "*")
 @RestController
 @RequestMapping("v1/project/users")
 public class UserController {
+    @Autowired
+    private UserDbService service;
+    @Autowired
+    private UserMapper userMapper;
+
+    @RequestMapping(method = RequestMethod.GET, value = "getUsers")
+    public List<UserDto> getUsers() {
+        return userMapper.mapToUserDtoList(service.getAllUsers());
+    }
 
-    @RequestMapping(method = RequestMethod.POST, value = "createUser")
-    public void createUser(UserDto userDto) {
+    @RequestMapping(method = RequestMethod.GET, value = "getUser")
+    public UserDto getUser(@RequestParam long id) throws UserNotFoundException {
```

Comment:
może nie być tutaj throws

Reply:
A czy to nie jest tak, że skoro w UserRepository jest:
`@Override
    Optional<User> findById(Long id);`
w UserController metoda getUser powinna wyrzucać wyjątek?
Metoda createUser nie wyrzuca błędu.